RubberduckTests/VBEditor/DerivedComSafeTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using Rubberduck.VBEditor.ComManagement;
+
+namespace RubberduckTests.VBEditor
+{
+    [TestFixture()]
+    public class StrongComSafeTests : ComSafeTestBase
+    {
+        protected override IComSafe TestComSafe()
+        {
+            return new StrongComSafe();
+        }
+    }
+
+    [TestFixture()]
+    public class WeakComSafeTests : ComSafeTestBase
+    {
+        protected override IComSafe TestComSafe()
+        {
+            return new WeakComSafe();
+        }
+    }
```

Comment:
could this not be accomplished through data-driven tests? That might make a tad more sense and keeps the different possibilities in one file...

Reply:
I think it could be rewritten using data-driven tests. However, that would make it semantically a bit different: With the current approach, the derived test classes contain all tests of the abstract one; with data driven tests, the selection for which derived classes to run the tests would be made in the attribute on top of each individual test. 

Admittedly, the `TestCaseSource` could be one method in the `TestFixture`, but it would have to be repeated on top of each test. As an alternative, I could simply move the derived test classes into the same file.